api/v1beta1/foundationdbcluster_types.go

```diff
@@ -157,6 +157,9 @@ type FoundationDBClusterSpec struct {
 	// client compatibility when performing an upgrade.
 	IgnoreUpgradabilityChecks bool `json:"ignoreUpgradabilityChecks,omitempty"`
 
+	// Buggify defines settings for injecting faults into a cluster for testing.
+	Buggify BuggifyConfig `json:"buggify,omitempty"`
```

Comment:
Do you expect to use this in the integration tests or more in (manual) e2e tests? If we want to use this for e2e tests I would rather look into https://chaos-mesh.org or something similar to reduce the complexity in the code.

Reply:
I think we want to be able to use this in integration tests, though I added it in this PR for the purpose of manual testing.